calculator.js

```diff
@@ -0,0 +1,157 @@
+
+i = 0;
+while (i < 1) { //loop that will continue running until a single calculation is completed
+  var typeOfCalculator = prompt("Which calculator would you like to use (b)basic, (a)advanced, bmi or trip?")
+  if (typeOfCalculator == "basic" || typeOfCalculator == "b") {
+    var value1 = parseInt(prompt('enter first value'));
+    var value2 = parseInt(prompt('enter the second value'));
+    var choice = prompt('Choose your action (a)add (s)subtract (m)multiply (d)divide (mod)modulus');
+    console.log(value1);
+    console.log(value2);
+
+      //basic calculator
+      switch (choice) {
+
+        //addition
+        case "a":
+        case "add" :
+          var value3 = value1 + value2;
+          alert(value3);
```

Comment:
This is still a little abrupt, I would at least include a message such as "The result is: " to make it a little more user friendly.